svelte-kit-scss/src/lib/components/shared/buttons/ButtonText.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+  export let text = String();
+</script>
+
+<button class="button-text">
+  {#if $$slots.left}
+    <span><slot name="left" /></span>
+  {/if}
+
+  {#if text}
+    <span>{text}</span>
```

Comment:
Why have the `text` prop at all?

Reply:
`<ButtonText text={sometext} />` vs `<ButtonText>{someText}<ButtonText>` isn't a big difference.
I think you should pick one (and pick the slot one imo) and go with it